ddprof-lib/src/main/cpp/javaApi.cpp

```diff
@@ -68,6 +68,11 @@ Java_com_datadoghq_profiler_JavaProfiler_init0(JNIEnv *env, jclass unused) {
   return VM::initProfilerBridge(nullptr, true);
 }
 
+extern "C" JNIEXPORT jboolean JNICALL
+JavaCritical_com_datadoghq_profiler_JavaProfiler_init0() {
+  return VM::initProfilerBridge(nullptr, true);
+}
+
```

Comment:
I'm curious, why have a `JavaCritical` variant for ìnit0`? This one doesn't look like it would ever be on the critical path so I don't see when the JVM would pick this one? :eyes: